cmd/install/install.go

```diff
@@ -40,49 +41,35 @@ var (
 )
 
 func NewCommand() *cobra.Command {
+	var (
+		toolsOnly bool
+		capiOnly  bool
+	)
 	command := &cobra.Command{
 		Use:               "install",
 		Short:             "Install required tools for Arlon",
 		Long:              "Install kubectl, Argocd cli, check git cli, and install compatible CAPI version on the management cluster",
 		DisableAutoGenTag: true,
 		Example:           "arlon install --kubectlPath <string> --argocdPath <string> --kubeconfigPath /path/to/kubeconfig",
 		RunE: func(c *cobra.Command, args []string) error {
-			fmt.Println("Note: SUDO access is required to install the required tools for Arlon")
-			fmt.Println()
-			var err error
 			// Install kubectl and point it to the kubeconfig
-			_, err = installKubectl()
-			if err == ErrKubectlPresent {
-				fmt.Println(Green("✓") + " kubectl is already present at default(" + Red(defaultKubectlPath) + ")location or user specifed location")
-			} else if err != nil {
-				fmt.Println(Red("x ")+"Error while installing kubectl ", err)
-			} else {
-				fmt.Println(Green("✓") + " Successfully installed kubectl")
-			}
-
-			fmt.Println()
-			_, err = verifyGit()
-			if err == ErrGitPresent {
-				fmt.Println(Green("✓") + " git is already present in the path")
-			} else {
-				fmt.Println(Yellow("! ") + "Install git cli")
+			isCapiOnly, _ := strconv.ParseBool(c.Flag("capi-only").Value.String())
+			isToolsOnly, _ := strconv.ParseBool(c.Flag("tools-only").Value.String())
+			if !isToolsOnly && !isCapiOnly { // none of these flags were set, so set to true and install both
```

Comment:
Is there a need to add these flags to the example command above or are they only for making some internal development easier ?

Reply:
Both. UX and for developers- internal and external.